doc/src/build/move.md

```diff
@@ -161,9 +161,9 @@ and [structs](https://github.com/move-language/move/blob/main/language/documenta
 in the Move book.
 
 In order for a Move struct type to define a Sui object type such as
-`Coin`, its first field must be `id: VersionedID`, which is a
+`Coin`, its first field must be `info: Info`, which is a
 struct type defined in the
-[ID module](https://github.com/MystenLabs/sui/blob/main/crates/sui-framework/sources/id.move). The
+[object module](https://github.com/MystenLabs/sui/blob/main/crates/sui-framework/sources/object.move). The
```

Comment:
Is this file being renamed?:
[object module](https://github.com/MystenLabs/sui/blob/main/crates/sui-framework/sources/object.move)

It doesn't exist now.

Reply:
> Correct, it's a renamed module after this PR lands

Hi Todd,

I missed a couple and am addressing them in:
https://github.com/MystenLabs/sui/pull/3300
https://github.com/MystenLabs/sui/pull/3301

Please take a look when you get a chance. Thanks!